keyboards/ktec/ergodone/keymaps/art/keymap.c

```diff
@@ -1,68 +1,52 @@
+// Copyright 2022 Artjoms Rizihs (@artjomsR)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 #include "art.h"
 #include "sendstring_workman_zxcvm.h"
-
-bool is_win = true;
+#include "user_config.c"
```

Comment:
c files need to be added by SRC, not included, like this.

Reply:
should be fixed now, same for all the rest